lib/contestant.rb

```diff
@@ -0,0 +1,24 @@
+class Contestant
+  attr_reader :age, :state_of_residence, :spending_money, :game_interests
+  def initialize(argument)
+    @first_name = argument[:first_name]
+    @last_name = argument[:last_name]
+    @age = argument[:age]
+    @state_of_residence = argument[:state_of_residence]
+    @spending_money = argument[:spending_money]
+    @game_interests = []
+  end
+
+  def full_name
+    @first_name + " " + @last_name
+  end
+
+  def out_of_state?
+    return false if @state_of_residence == "CO"
```

Comment:
This is close, but there i no way that a true value can ever be returned here. If you add a `true` as the final line of the method, then this would work.